samcli/lib/samlib/wrapper.py

```diff
@@ -77,6 +81,30 @@ def run_plugins(self, convert_local_uris=True):
     def template(self):
         return copy.deepcopy(self._sam_template)
 
+    def _patch_language_extensions(self):
```

Comment:
Can we mention here that we are temporarily disabling validation for `DeletePolicy` and `UpdateReplacePolicy` for language extensions?